src/main/java/org/springframework/data/couchbase/transaction/CouchbaseResourceHolder.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Currently holds only the CoreTransactionAttemptContext.
In the future, it could also hold a map of getResults for the current transaction, such that subsequent operation could lookup the TransactionResult and use that instead of refetching with a ctx.get().